maestro-app/src/main/java/bio/overture/maestro/app/infra/config/properties/DefaultApplicationProperties.java

```diff
@@ -237,6 +244,13 @@ public int maxDataLength() {
     private StorageType storageType = StorageType.S3;
   }
 
+  @Data
+  @ToString
+  @EqualsAndHashCode
+  private static class WebClientConfig {
```

Comment:
the name of the class should match the yaml, i.e. this shouldbe `Webclient` you can test that your configuration goes through or not

Reply:
like try changing it in app yaml to something small like 200 and fetching a bigger study will fail